saleor/order/tasks.py

```diff
@@ -45,28 +45,24 @@ def send_order_updated(order_ids):
 
 
 def _bulk_release_voucher_usage(order_ids):
-    # code commented to fix mypy issue, this will be fixed in separate PR with
-    # expire_orders_task adjustments
-
-    # voucher_orders = Order.objects.filter(
-    #     voucher=OuterRef("pk"),
-    #     id__in=order_ids,
-    # )
-    # count_orders = voucher_orders.annotate(
-    #     count=Func(F("pk"), function="Count")
-    # ).values("count")
-
-    # Voucher.objects.filter(
-    #     Exists(voucher_orders),
-    #     usage_limit__isnull=False,
-    # ).annotate(
-    #     order_count=Subquery(count_orders)
-    # ).update(used=F("used") - F("order_count"))
+    voucher_orders = Order.objects.filter(
+        voucher_code__voucher=OuterRef("voucher_id"),
```

Comment:
`voucher_code__voucher` should be replace with `Exsists` as it's doing `join` on the tables

Reply:
fixed